src/loaders/svg/tvgSvgSceneBuilder.cpp

```diff
@@ -199,6 +199,27 @@ static unique_ptr<RadialGradient> _applyRadialGradientProperty(SvgStyleGradient*
 }
 
 
+static bool _appendScene(SvgNode* node, Scene* scene, float vx, float vy, float vw, float vh)
```

Comment:
I hope that this operation also considers the clip-path.
(Of course I know that this should be solved first.(https://github.com/Samsung/thorvg/issues/524)......)

Reply:
no, it does not, since the clipPath is not a container element (https://www.w3.org/TR/SVG11/intro.html#TermContainerElement):

"container element - An element which can have graphics elements and other container elements as child elements. Specifically: ‘a’, ‘defs’, ‘glyph’, ‘g’, ‘marker’, ‘mask’, ‘missing-glyph’, ‘pattern’, ‘svg’, ‘switch’ and ‘symbol’."

so according to the svg standard a clip can not be a scene (on the svg_loader site, as I understand, the api can work as we want, so #661 can still be applied)

Reply:
Oh I didn't know this spec at all. Thanks for URL :)